scripts/staging-accounts.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
we don't need to export the private keys (while they _are_ from a private chain) since we only need the addresses themselves, we can simply export an array of addresses